app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="blue">
+  <div class="container-sm">
+    <div class="header-container blue d-flex justify-between align-end">
+      <a href="<%= categories_path %>" class="back-button"><i class="fa-solid fa-arrow-left" style="color: #ffffff;"></i></a>
+      <h1 class="category-title"><%= @category.name %></h1>
+      <a href="<%= categories_path %>" class="btn-login blue"><i class="fa-solid fa-magnifying-glass" style="color: #ffffff;"></i></a>
+    </div>
+    <div class="d-flex justify-between gap-10">
+            <div class="w-1-3">
+
+            </div>
+
+            <div class="small-div w-1-3 d-flex flex-column align-center justify-end most-recent-container ">
+              <p class="text">Most recent</p>
+              <p class="dot">•</p>
+            </div>
+            <div class="small-div w-1-3 d-flex flex-column align-center justify-end most-ancient-container">
+              <p class="text">Most ancient</p>
+              <p class="dot">•</p>
+            </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Kindly fix your design in terms of font weight, and color for the categories text. Notice that the text for the price is bold and the color is #434b54.

Your design
![image](https://github.com/ClaudiaRojasSoto/BudgetApp/assets/81932478/d71a31ea-4d8f-41fa-8ea2-e81a21eb4173)

Given design
![image](https://github.com/ClaudiaRojasSoto/BudgetApp/assets/81932478/8652c60d-1f23-42a3-af21-fb32c8aa9150)

Reply:
Done @Munsa1 Thank you